lib/render/content.php

```diff
@@ -126,16 +126,4 @@ function RenderAOTWComponent($achID, $forumTopicID): void
 
     echo "</div>";
     echo "</div>";
-}
-
-function RenderConsoleMessage(int $consoleId): void
-{
-    // PS2
-    if ($consoleId === 21) {
-        echo <<<HTML
-            <div class="mb-4">
-                <a href="/viewtopic.php?t=11108">⚠️️ Achievement developers are currently involved in a PlayStation 2 rollout. The planned launch date is October 1st, 2022 at 00:00 GMT! Click for more details. ⚠️</a>
-            </div>
-        HTML;
-    }
-}
+}
```

Comment:
The CI is complaining about this (no newline at end of file). I thought the git hooks were supposed to prevent that.

Reply:
This threw me off because all I did was reverse the change from commit #985 - I didn't add anything to it.

https://github.com/RetroAchievements/RAWeb/blob/d74a02e00e4cd3116682858ad26048b6b3863f51/lib/render/content.php#L129

There's not a new line at the end; there's only a bracket 😭 
![wat](https://user-images.githubusercontent.com/45054151/192898038-84e52aa5-cb64-402d-957c-fd9074d45873.PNG)